pkg/trust/trust.go

```diff
@@ -167,84 +175,127 @@ func CreateTmpFile(dir, pattern string, content []byte) (string, error) {
 	return tmpfile.Name(), nil
 }
 
-// GetGPGId return GPG identity, either bracketed <email> or ID string
-// comma separated if more than one key
-func GetGPGId(keys []string) string {
+func getGPGIdFromKeyPath(path []string) []string {
+	var uids []string
+	for _, k := range path {
```

Comment:
(this does work fine, but the caller already has a loop.  Instead of the caller building a list of paths, and then calling `getGPGIdFromKeyPath` on that list, the caller could just call `getGPGIdFromKeyPath` in each iteration of its loop, and collect the data into the `uids` array directly.

BTW when `path` is an array, both `path` and the function name …`FromKeyPath` are named in a misleading way.)